### DISCUSSION TITLE
Some things don't work as expected...

### DISCUSSION THREAD

**Discussion by ben2talk** [zaps166/QMPlay2#658] [Ideas]
Having used several players, there are amazing functions not common which is very sad.

1. Guayadeque - great drag/drop and movable window elements (including library and last.fm fusion). The first player (not so good now) to get lyrics reliably, very useful 'search' and also 'search in browser' buttons on lyrics pane. Also can use context/edit and paste and edit before saving right into the tags.

3. Strawberry - again, excellent multiple playlist function with drag/drop functionality to 'move/copy/open with' any entry. Strawberry is great at grabbing lyrics, and if it can't find them, you can simply paste and edit your own text there.

4. DeaDBeef - incredible ability to right click and do amazing things. This is a primary one for me to load up some DSD or FLAC or cue-flac to load a list of tracks, and export via conversion to Opus --192 (DIY entry, though sane defaults are decent enough).

QMPlay would be a great tool to load a freshly imported selection of files and work this way - I love the Information display (best of all - also the 'simple visualisation' showing activity on all channels (my music is mostly 2, but many are also 4/5/6 channels as are videos) though Deadbeef does a sterling job of listing playback data in the bottom of the window).

So the first issue I met with QMPlay2 (it loads great, plays great, looks great) is that after embedding lyrics (which work in other players) often Lyrics won't show and there's no option to edit/paste from a web search. No options, either, to use context to open with another application.

Drag and Drop fails (e.g. drag a track to Musicbrainz to scan and move it to the library).

**Comment by zaps166** [comment]
Could you explain the lyrics?

D&D should be possible by middle click - could you try it?

**Comment by ben2talk** [comment]
Ok, you're right - that's very strange that it works with middle click!!! Thanks for that.

For lyrics, generally it's easy in Strawberry, otherwise I generally paste from clipboard.
![Drag Lyrics](https://github.com/zaps166/QMPlay2/assets/6983154/59af9247-fb8c-4026-97fe-166fe060efa5)

**Reply by zaps166** [reply]
Left click is doing DnD on playlist only, middle click can copy file or move to another player.

If lyrics is a feature request, please open a feature request and make a full description - I don't use Strawberry and idk how it works.
